content/1.7/back-office/catalog/stock/stock.md

```diff
@@ -0,0 +1,31 @@
+
+---
+title: Stock
+weight: 1
+---
+
+  
+# SPECS Stock & Stock movement
+
+## Stock
+
+  
+| Stock's type | Definition | When impacted | 
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Physical stock | Number of products that are physically in my stock.| When a merchant can add or remove physical stock 
```

Comment:
IMHO we can't list every use case next to the definition, but user stories should provide the details.
```suggestion
| Physical stock | Number of products that are physically in my stock.| When a merchant can add or remove physical stock, when ordered products leave or return in stock, ...
```